kart/repo.py

```diff
@@ -584,8 +585,8 @@ def head_tree(self):
 
     @property
     def empty_tree(self):
-        """Returns the empty tree, with SHA 4b825dc642cb6eb9a060e54bf8d69288fbee4904."""
-        return self['4b825dc642cb6eb9a060e54bf8d69288fbee4904']
+        f"""Returns the empty tree, with SHA {EMPTY_TREE_SHA}."""
+        return self[EMPTY_TREE_SHA]
```

Comment:
isn't this the same as `KartRepo.EMPTY_TREE`?

Reply:
Yes... removed one of them.